packages/commonwealth/client/scripts/views/components/comments/edit_comment.tsx

```diff
@@ -45,38 +80,12 @@ export const EditComment = (props: EditCommentProps) => {
           label="Cancel"
```

Comment:
![image](https://user-images.githubusercontent.com/14819225/225888448-62310e37-da6a-47db-806e-e34b03b0f936.png)

When I write new comment, cancel button dose nothing. Maybe we can hide it when writing the comment and show only when we edit it

Reply:
Made it so that the cancel button clears the text– hidden if there is no text.